src/Core/Prefab/GlobeView.js

```diff
@@ -193,7 +193,7 @@ function GlobeView(viewerDiv, coordCarto, options = {}) {
     const sun = new THREE.DirectionalLight();
     sun.position.set(-0.5, 0, 1);
     sun.updateMatrixWorld(true);
-    this.scene.add(sun);
+    wgs84TileLayer.object3d.add(sun);
```

Comment:
The sun, atmosphere object doesn't really make sense in the layer wgs84TileLayer
It would be necessary to create a new geometric layer for these objects.